src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -1957,5 +1963,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         FEATURE_ImplementingReadonlyOrWriteonlyPropertyWithReadwrite
         FEATURE_DigitSeparators
         FEATURE_BinaryLiterals
+        FEATURE_Tuples
```

Comment:
This feature flag isn't much used in this PR. Do you plan to add diagnostics to the syntax tree when the feature flag isn't set?